packages/pwa-storefront-mcp/src/server/server.test.js

```diff
@@ -37,7 +37,12 @@ describe('PwaStorefrontMCPServerHighLevel integration', () => {
     it('should list registered tools via stdio', async () => {
         const child = spawn(BABEL_NODE_PATH, ['src/server/server.js'], {
             cwd: process.cwd(),
-            stdio: ['pipe', 'pipe', 'inherit']
+            stdio: ['pipe', 'pipe', 'inherit'],
+            env: {
+                ...process.env,
+                // NOTE: THIS ENV VAR IS  USUALLY SET BY CURSOR OR THE MCP SERVER?
+                WORKSPACE_FOLDER_PATHS: path.resolve(process.cwd(), '..', '..')
+            }
```

Comment:
WORKSPACE_FOLDER_PATHS env var is normally added by the server or ide I believe. Since we are running the server directly it's not in the env list so we are adding our own mock value.